lib/liquid/context.rb

```diff
@@ -73,10 +73,11 @@ def pop_interrupt
       @interrupts.pop
     end
 
-    def handle_error(e, line_number = nil)
-      if e.is_a?(Liquid::Error)
+    def handle_error(e, line_number = nil, line = nil)
```

Comment:
Ah sorry, line is a bad variable name, I was wanting that to be line, as in the raw liquid code on that line.